src/pages/components/button/usage.mdx

```diff
@@ -13,17 +13,46 @@ Buttons are used to initialize an action. Button labels express what action will
 <AnchorLinks>
 
 <AnchorLink>Overview</AnchorLink>
-<AnchorLink>Button order</AnchorLink>
-<AnchorLink>Button alignment</AnchorLink>
-<AnchorLink>Labels</AnchorLink>
-<AnchorLink>Icon usage</AnchorLink>
-<AnchorLink>Danger button usage</AnchorLink>
-<AnchorLink>Variations</AnchorLink>
+<AnchorLink>Live demo</AnchorLink>
+<AnchorLink>Formatting</AnchorLink>
+<AnchorLink>Content</AnchorLink>
+<AnchorLink>Modifiers</AnchorLink>
+<AnchorLink>Related</AnchorLink>
 <AnchorLink>References</AnchorLink>
 <AnchorLink>Feedback</AnchorLink>
 
 </AnchorLinks>
 
+## Overview
+
+Buttons are clickable elements that are used to trigger actions. They communicate calls to action to the user and allow users to with pages in a variety of ways. Button labels express what action will occur when the user interacts with it.
+
+#### When to use
+
+Buttons are clickable elements that are used to trigger actions. They communicate calls to action to the user and allow users to with pages in a variety of ways. Button labels express what action will occur when the user interacts with it.
```

Comment:
```suggestion
Use buttons to communicate actions users can take and allow them to interact with the page. Each page should have one primary button, and any remaining calls-to-action should be represented as lower emphasis buttons.
```
I think this paragraph should be different than the overview above. I wrote this up quickly so feel free to take or leave the suggested content 🙂